.mvn/extensions.xml

```diff
@@ -3,7 +3,7 @@
   <extension>
     <groupId>com.gradle</groupId>
     <artifactId>develocity-maven-extension</artifactId>
-    <version>1.23</version>
+    <version>1.22.2</version>
```

Comment:
@clayburn why do we need a downgrade here?

Reply:
The current version of the server `2024.2.7` does not support version `1.23` of the extension. I think it happened to work, but strictly it is not supported until the Develocity server is on 2024.3.x.

Once this migration is complete to develocity.apache.org, upgrades should happen more quickly such that updating the extension before the server is not a concern.